packages/db/src/sql/pg.ts

```
@@ -1,289 +1,253 @@
-            logger.error('Unable to load pg module', e)
-            reject(e)
-          })
+            logger.error("Unable to load pg module", e);
+            reject(e);
+          });
       }
-    })
+    });
   }
 
   rawQuery(query: string): Promise<any> {
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)